marv/app/listing.py

```diff
@@ -251,6 +257,12 @@ def fileset_listing():
 
         site = current_app.site
         query = Listing.query
+        if not filters:
+            #Only show recent stuff if no filter is given
+            time = datetime.today() - timedelta(days=2)
+            date = time.strftime("%Y-%m-%dT00:00:00-04:00");
```

Comment:
Why add a time zone offset? Use 'date time.datetime.utcnow()' in the line above it should always make it two days in UTC.